edk2toollib/utility_functions_test.py

```diff
@@ -91,5 +149,10 @@ def test_EnumClass_args(self):
         # check to make sure the values are unique
         self.assertNotEqual(colors.Green, colors.Red)
 
+
 # DO NOT PUT A MAIN FUNCTION HERE
 # this test runs itself to test runpython script, which is a tad bit strange yes.
+test = UtilityFunctionsTest()
```

Comment:
Adding these tests would seem to be in contradiction to the above note, no?

Reply:
Yes- pytest on my dev box hangs so I run the tests manually. This is me being dumb and pushing up without checking carefully